indico/MaKaC/common/output.py

```diff
@@ -1365,26 +1351,26 @@ def resourceToXMLMarc21(self, res, out=None):
             self.resourceLinkToXMLMarc21(res, out=out)
 
     def _attachment_unique_id(self, attachment, add_prefix=True):
-        unique_id = uniqueId(attachment.folder.linked_object)
+        unique_id = str(uniqueId(attachment.folder.object))
```

Comment:
i'd rather fix the code two lines below and use `'INDICO.{}'.format(unique_id)` there